docs: make example fixtures module-scoped, explain Python 3.12 requirement (#65)
canonical/jubilant: docs/tutorial/getting-started.md

    assert status.apps['snappass-test'].scale == 1
```

> You may want to adjust the [scope](https://docs.pytest.org/en/stable/how-to/fixtures.html#fixture-scopes) of your `juju` fixture. For example, if you want to create a new model for every test function, use `scope='function'`, or just omit the scope (function scope is pytest's default):

Reviewer: I'd probably simplify this slightly:

```suggestion
You may want to adjust the [scope](https://docs.pytest.org/en/stable/how-to/fixtures.html#fixture-scopes) of your `juju` fixture. For example, to create a new model for every test function (pytest's default behavior), omit the scope:
```

Author: Good call, done
